lam4doc/services/helpers.py

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/python3
+
+# helpers.py
+# Date:  23/12/2020
+# Author: Mihai Coșleț
+# Email: coslet.mihai@gmail.com 
+
+import logging
+from collections import namedtuple
+
+from lam4doc.config import LAM_LOGGER
+
+logger = logging.getLogger(LAM_LOGGER)
+
+FileInfo = namedtuple('FileInfo', 'location file_name')
```

Comment:
looks like a config of some sort; if so move it to the configs then and void this module

Reply:
it doesn't feel natural for me to define it as a config. it's a helper structure that allows using tuples, but not having to write `tuple[0]` and `tuple[1]`, as it hurts the readability of the code.

I don't think it is a domain level model either. 

I'd leave it here, for now, and see how the codebase evolves, and make the decision then (as it involves just moving the definition of the namedtuple)